CHANGELOG.md

```diff
@@ -2,12 +2,17 @@
 
 All notable changes to this project will be documented in this file.
 
-## [4.4.4] - TBD
+## [4.4.5] - TBD
 
-Wazuh commit: TBD
-Release report: TBD
+Wazuh commit: TDB
+Release report: TDB
 
-## Added
+## [4.4.4] - 13-06-2023
+
+Wazuh commit: https://github.com/wazuh/wazuh/commit/32b9b4684efb7c21ce71f80d845096549a5b4ed5
+Release report: https://github.com/wazuh/wazuh/issues/17520
+
+### Added
```

Comment:
We need to change this

The changelog should refer to wazuh-qa repository, not to wazuh repository, the wazuh repository has its own changelog file.

We also need to stop referring to RC issues, the wazuh-qa changelog should have changes related to wazuh-qa repository, as wazuh-packages does

I propose the following format to be applied from now, maintaining the older changelog for history:

```
## [4.4.5]

### Added
### Fixed
```

Reply:
Agree to any possible improvement on our SDLC ! but this is currently urgent and this is the way CHANGELOG is being handled since 4.2.0